claimedtasks.md

```diff
@@ -57,8 +57,42 @@ This is a **read-only** endpoint, the [POST to the claimed task](#post-method-si
 
 It returns the tasks claimed by the specified user
 
-## POST Method (collection level)
-The creation of claimed tasks is managed by the underline workflow system. No methods are exposed to manually trigger such creation to avoid workflow hjack and inconsistency.
+#### findAllByItem
+**/api/workflow/claimedtasks/search/findAllByItem?uuid=<:item-uuid>**
+It returns all the claimed tasks related to the specified item
+
+The supported parameters are:
+* page, size [see pagination](README.md#Pagination)
+* uuid: mandatory, the uuid of the item object
+
+Return codes:
+* 200 OK - if the operation succeed
+* 401 Unauthorized - if you are not authenticated
+* 403 Forbidden - if you are not logged in with sufficient permissions. Only users with ADMIN right can use the endpoint
+
+#### findByItem
+**/api/workflow/claimedtasks/search/findByItem?uuid=<:item-uuid>**
+It returns, if any, the single task related to the specified item claimed by the current user
+
+The supported parameters are:
+* page, size [see pagination](README.md#Pagination)
+* uuid: mandatory, the uuid of the item object
+
+Return codes:
+* 200 OK - if the operation succeed
+* 204 No Content - if there is no claimed task for the specified item and the current user
+* 401 Unauthorized - if you are not authenticated
+
+## POST Method
+To create a claimedtask, i.e. to clam a poooltask. 
```

Comment:
Minor typo: should be `pooltask`

Reply:
Very minor...you have an extra `o` here...it says `poooltask` which is a misspelling of `pooltask`